### PR TITLE
feat(icons): create subtract icon LS-3892

### PR DESCRIPTION
**Pull request by ryanjdiercks**
## Jira

https://littlespoon.atlassian.net/browse/LS-3892

## Motivation

Create subtract icon in icons package

### CODE REVIEW

File: packages/icons/README.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ icons.CloseIcon
 icons.ExclamationIcon
 icons.FilterIcon
 icons.OpenIcon
+icons.SubtractIcon
```
Comment (remarkablemark) [inline packages/icons/README.md:33]: My bad for not mentioning in kickoff but what do you think about the name `MinusIcon`? @ryanjdiercks @fjallkraken

**Review by remarkablemark — COMMENTED** [review]
Left a comment but looks good overall!

File: packages/icons/README.md (hunk 1 above)
Comment (ryanjdiercks) [reply packages/icons/README.md:33]: @chantaldegaust  used `subtract` here.
https://zeroheight.com/3ddd0f892/p/699b01-icons/b/315972

Comment (remarkablemark) [reply packages/icons/README.md:33]: Got it, then let's keep it consistent

**Review by remarkablemark — APPROVED** [review]

File: packages/storybook/src/stories/icons/SubtractIcon.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+export const Large = Template.bind({})
+Large.args = {
+  size: 'large',
+}
+
+export const Fill = Template.bind({})
+Fill.args = {
+  fill: 'skyblue',
+}
```
Comment (remarkablemark) [inline packages/storybook/src/stories/icons/SubtractIcon.stories.tsx:37]: Does the fill look a little uncentered? https://deploy-preview-589--littlespoon.netlify.app/?path=/story/design-system-icons-subtracticon--fill

Screen Shot 2022-08-24 at 5 21 37 PM

Comment (ryanjdiercks) [reply packages/storybook/src/stories/icons/SubtractIcon.stories.tsx:37]: I think the svg file itself has some alignment issue.
I updated the svg file and pushed a new commit
Screen Shot 2022-08-25 at 12 04 28 PM

**Review by remarkablemark — APPROVED** [review]

**Comment by remarkablemark** [comment]
Passed desk check: https://deploy-preview-589--littlespoon.netlify.app/?path=/story/design-system-icons-subtracticon--fill

Good to merge @ryanjdiercks

**Pull request merged**